### PR TITLE
Some essential admin features

### PR DESCRIPTION
**Pull request by erfanio**
- [x] Get admin auth in slack
- [x] Allow admins see all active tokens (allows login without access to email)

#procrastinatinginexamperiod:no_good_man:

### CODE REVIEW

**Comment by erfanio** [comment]
Maybe the list of tokens should only be available in development environment? :thinking:

File: app/controllers/token.js [hunk 1]
```diff
@@ -1,68 +1,58 @@
...
-
-const { User, Token } = require('../models');
-const { create: createJwt } = require('../util/jwt');
+const service = require('../services/token');
+const constant = require('../constants');
 const env = require('../../env');
 
-const emailClient = new Postmark.Client(env.POSTMARK_CLIENT_KEY);
+exports.list = {
+  handler: async () => service.list(),
```
Comment (erfanio) [inline app/controllers/token.js]: add a `if (env.DEV)` here?

File: app/controllers/token.js [hunk 2]
```diff
@@ -1,68 +1,58 @@
...
+};
 
-exports.validate = {
+exports.request = {
   handler: async (req) => {
-    const { token } = req.payload;
+    const { email } = req.params;
+
+    // bypass normal auth for admin and send jwt to slack
+    if (email === 'admin@unihack') {
```
Comment (erfanio) [inline app/controllers/token.js]: This makes me feel kinda wrong... (the hard coded fake email i mean)

Maybe adding this directly on slack with `/tokenbot admin` is better hmmm....

**Comment by erfanio** [comment]
thoughts @unihackhq/backend

File: app/controllers/token.js (hunk 2 above)
Comment (terencehuynh) [reply app/controllers/token.js]: Should we make this a constant that's changed as an ENV variable?

**Pull request merged**